alpine/src/main/scala/io/delta/alpine/internal/storage/ReadOnlyLogStoreProvider.scala

```diff
@@ -17,10 +17,26 @@
 package io.delta.alpine.internal.storage
 
 import io.delta.alpine.ReadOnlyLogStore
+import io.delta.alpine.sources.AlpineHadoopConf
 import org.apache.hadoop.conf.Configuration
 
 private[internal] trait ReadOnlyLogStoreProvider {
-  def createLogStore(hadoopConf: Configuration): ReadOnlyLogStore =
-    // TODO read logStore class from HadoopConf
-    new HDFSReadOnlyLogStore(hadoopConf)
+  import ReadOnlyLogStoreProvider._
+
+  def createLogStore(hadoopConf: Configuration): ReadOnlyLogStore = {
```

Comment:
Can we move this method to `object ReadOnlyLogStore` and remove this file? We added `LogStoreProvider` because DBR and OSS Delta Lake need two different implementations. But in this project, we don't need it.